fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/MatchCandidate.java

```diff
@@ -261,6 +261,7 @@ default Set<String> getQueriedRecordTypeNames() {
     }
 
     @Nonnull
+    @SuppressWarnings("fallthrough")
```

Comment:
Removed. For reference, I added this while working on the PR when during an earlier version, the `PERMUTED_MIN` and `PERMUTED_MAX` cases added a value index expansion first and then fell through to the other aggregate index cases. But I've removed that fall through and now just specify both expansions in a case that is specific to the `PERMUTED_MIN` and `PERMUTED_MAX` index types because the other approach was a little too clever by half